lib/nessus/report_item.rb

```diff
@@ -87,7 +87,8 @@ def method_missing(method, *args)
       #   exploit_framework_metasploit, exploit_framework_core
       tag = @xml.xpath("./#{method_name}").first
       if tag
-        return tag.text
+        text = tag.text
+        return tags_with_html_content.include?(method) ? cleanup_html(text) : text
```

Comment:
```suggestion
        return method == :description ? cleanup_html(text) : text
```
We could save some lines by comparing directly here

Reply:
I will defer to you for any code optimization. I was just replicating the Netsparker setup!

Reply:
Got it, yeah I think since this only involves one specific tag, we can compare directly and avoid having an extra method

Reply:
I hear you @caitmich but I suspect this list is going to grow in the future, it could be the toss of a coin, but I think is not a bad thing to have this possibility of expansion factored in.